src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1375,87 +1360,69 @@ bool AdaptDispatch::ScrollDown(const VTInt uiDistance)
     return true;
 }
 
-// Routine Description:
-// - DECSCPP / DECCOLM Sets the number of columns "per page" AKA sets the console width.
-// DECCOLM also clear the screen (like a CSI 2 J sequence), while DECSCPP just sets the width.
-// (DECCOLM will do this separately of this function)
-// Arguments:
-// - columns - Number of columns
-// Return Value:
-// - True.
-bool AdaptDispatch::SetColumns(const VTInt columns)
-{
-    const auto viewport = _api.GetViewport();
-    const auto viewportHeight = viewport.bottom - viewport.top;
-    _api.ResizeWindow(columns, viewportHeight);
-    return true;
-}
-
 // Routine Description:
 // - DECCOLM not only sets the number of columns, but also clears the screen buffer,
 //    resets the page margins and origin mode, and places the cursor at 1,1
 // Arguments:
-// - columns - Number of columns
+// - enable - the number of columns is set to 132 if true, 80 if false.
 // Return Value:
-// - True.
-bool AdaptDispatch::_DoDECCOLMHelper(const VTInt columns)
+// - <none>
+void AdaptDispatch::_SetColumnMode(const bool enable)
```

Comment:
This is essentially just a merge of the `SetColumns` method (which was removed from `ITermDispatch`) and the `_DoDECCOLMHelper` method. But this also now tracks the current state of the `DECOLM` mode so it can be reported from `DECRQM`.